lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -326,6 +335,9 @@ def perform_module_state_update(update_fcn, filter_fcn, _entry_id, course_id, ta
 
     for module_to_update in modules_to_update:
         task_progress.attempted += 1
+        # find the problem descriptor:
+        module_descriptor = modulestore().get_item(module_to_update.module_state_key)
```

Comment:
how is module_descriptor used after fetching it. In the diffs, I don't see it references...